modules/service-esm.sh

```diff
@@ -1,6 +1,8 @@
 # shellcheck disable=SC2039
 
 ESM_SUPPORTED_SERIES="precise"
+ESM_SUPPORTED_ARCHS=""
```

Comment:
I think using a special key here is better than an empty string. How about "ALL", or "@all@", or something like that?

Reply:
"ALL" sounds good to me, as it matches deb archs. Done